bin/benchpark

```diff
@@ -174,22 +176,22 @@ def symlink_tree(src, dst):
 def benchpark_setup_handler(args):
     """
     workspace_root/
```

Comment:
We should think of a different names than `workspace_root/` and `workspace/` (or at least, one of those two should change): for example we could have `experiments_basedir/` as the root and `experiment_workspace/` for each Ramble workspace.

That would improve our docs in general, including possibly simplifying the wording of docs added in this PR.

Reply:
Yes, I am also convinced that this is confusing.

Ramble uses "workspace" in its docs, so I think we should keep using the term "workspace" to mean Ramble workspace for a single experiment.  It will be nested within the experiment triple, so I am not sure we need to add experiment_.

"Benchpark workspace" or "workspace_root" then need to not have the word "workspace" in it.  I like your suggestion of experiments_basedir, let me try to guess what you were trying to convey:
- I like that it implies multiple experiments, but wonder if a single letter "s" is enough for this to be visible?  
- By basedir, are you trying to convey that the directory contains the set of experiments?  And that the user ultimately gets to name it?

I think we can riff on one of the 2 themes here:

- It is a workspace in the sense Ramble is using it.  Are there antonyms we can use?  Preferably ones that don't have the word "work" or "space" in it?  With benchpark being an outside space I am not fond of indoor space names here (studio/study/office/shop) but is there an outdoorsy equivalent that would be fitting here?  What are your thoughts on: 
-- plant (too far into the tree analogy?)
-- mill (ok maybe that's mean to the trees in benchpark)
-- yard (probably my favorite atm)
-- studio (j/k, too close to VS)
-- branch (too far into the tree analogy?)

- Alternatively, we could opt for indicating more prominently that multiple experiments can live here.  I am hesitant to call it a suite because that has other connotations and I think we will use the term suite in another place.  So maybe a set_of_experiments or similar?